examples/driving_in_traffic/setup.py

```diff
@@ -0,0 +1,26 @@
+from os import path
+
+from setuptools import find_packages, setup
+
+this_dir = path.abspath(path.dirname(__file__))
+with open(path.join(this_dir, "README.md"), encoding="utf-8") as f:
+    long_description = f.read()
+
+setup(
+    name="driving-in-traffic",
+    description="Driving in traffic using DreamerV2",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    version="0.1.0",
+    packages=find_packages(),
+    include_package_data=True,
+    zip_safe=True,
+    python_requires=">=3.7",
+    install_requires=[
+        "setuptools>=41.0.0,!=50.0",
+        "smarts[camera-obs] @ git+https://git@github.com/huawei-noah/SMARTS@develop",
```

Comment:
Interesting, I have not seen this syntax before. it looks useful.

Reply:
This needs to be replaced with a fixed smarts version, once we have a smarts release.